frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx

```diff
@@ -30,7 +30,7 @@ const CreateProjectListPage: React.FC<CreateProjectListPageProps> = ({
     <ProjectListPage {...props} title={title}>
       {children} {t('devconsole~or')}{' '}
       <Button isInline variant="link" onClick={openProjectModal}>
-        {t('devconsole~Create a Project')}
+        {t('devconsole~create a Project.')}
       </Button>
```

Comment:
This string should be translated as a single translation and not a concatenation. You can use the `<Trans` component.

Reply:
fixed